tensorflow/core/distributed_runtime/rpc/grpc_worker_service.cc

```diff
@@ -337,32 +337,25 @@ void GrpcWorker::RecvTensorAsync(CallOptions* opts,
             if (src_dev->tensorflow_gpu_device_info() && (!on_host)) {
 #if GOOGLE_CUDA
               const DeviceContext* send_dev_context = send_args.device_context;
-              RecvTensorResponse* tmp = new RecvTensorResponse;
-              tmp->set_is_dead(is_dead);
+              AllocatorAttributes alloc_attrs;
```

Comment:
We don't have any benchmarks to measure the effect of this particular change. Do you want to take a stab at modifying rpcbench_test.cc to (optionally) allocate tensors on GPU? If not, I'll find time later on.

It's hard to know what impact this will have unless we measure it :)